lms/templates/emails/reverification_processed.txt

```diff
@@ -0,0 +1,43 @@
+<%namespace file="../main.html" import="stanford_theme_enabled" />
+<%! from django.utils.translation import ugettext as _ %>
+% if status == "approved":
+    ${_("Your verification for course {course_name} and assessment {assessment} "
+        "has been passed."
+    ).format(course_name=course_name, assessment=assessment)}
+
+%else:
+    ${_("Your verification for course {course_name} and assessment {assessment} "
+        "has failed."
+    ).format(course_name=course_name, assessment=assessment)}
+
+    % if not is_attempt_allowed:
+        ${_("You have reached your allowed attempts. No more retakes allowed.")}
+    % elif not verification_open:
+        ${_("Assessment date has passed and retake not allowed.")}
+    % else:
+        % if due_date:
+            ${_("Assessment closes on {due_date}.".format(due_date=due_date))}
```

Comment:
How are we formatting this due date?  There are some functions in `date_utils.py` that we probably want to use to ensure that the date format is localized.